kuka_external_control_sdk/kss/krl/Program/EKIserver/command_handler.src

```diff
@@ -0,0 +1,196 @@
+&ACCESS R
+DEF command_handler()
+    ; Software License Agreement (BSD License)
+    ;
+    ; Copyright (c) 2025, KUKA Hungaria Kft.
+    ; All rights reserved.
+    ;
+    ; Redistribution and use in source and binary forms, with or without
+    ; modification, are permitted provided that the following conditions are met:
+    ;
+    ;      * Redistributions of source code must retain the above copyright
+    ;        notice, this list of conditions and the following disclaimer.
+    ;      * Redistributions in binary form must reproduce the above copyright
+    ;        notice, this list of conditions and the following disclaimer in the
+    ;        documentation and/or other materials provided with the distribution.
+    ;      * Neither the name of the copyright holder, nor the names of its
+    ;        contributors may be used to endorse or promote products derived
+    ;        from this software without specific prior written permission.
+    ;
+    ; THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+    ; AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+    ; IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+    ; ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+    ; LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+    ; CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+    ; SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+    ; INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+    ; CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+    ; ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    ; POSSIBILITY OF SUCH DAMAGE.
+END
+
+GLOBAL DEF ROS_HandleCommand(cmd: IN)
+    DECL Command cmd
+    DECL CommandType cmd_enum
+
+    cmd_enum = ROS_StringToCommandType(cmd.cmd_str[])

```

Comment:
is there a reason for transferring this enum as a string? it might be more optimal to just use enums on both sides